django_evaluation/settings/local.py

```diff
@@ -326,6 +328,24 @@ def _set_favicon(html_color: str, project_root: Path) -> None:
             {"name": title, "url": reverse_lazy(url), "html_id": html_id}
         )
 
+OIDC_RP_CLIENT_ID = os.environ.get("OIDC_RP_CLIENT_ID", "freva")
```

Comment:
Could we stick to the naming convention in freva-rest `OICD_CLIENT_ID` and `OICD_CLIENT_SECRET`